aiohttp/cookiejar.py

```diff
@@ -165,8 +177,10 @@ def update_cookies(self,
             if max_age:
                 try:
                     delta_seconds = int(max_age)
-                    self._expire_cookie(self._loop.time() + delta_seconds,
-                                        domain, name)
+                    self._expire_cookie(datetime.datetime.now(
+                        datetime.timezone.utc) +
+                        datetime.timedelta(seconds=delta_seconds),
+                        domain, name)
```

Comment:
You need to handle OverflowError here. A malicious or broken server could send max_age=_more seconds than time available in the universe_.